entitlement/src/main/java/org/killbill/billing/entitlement/api/DefaultEntitlementContext.java

```diff
@@ -32,9 +35,7 @@
 import org.killbill.billing.util.callcontext.CallOrigin;
 import org.killbill.billing.util.callcontext.UserType;
 
-import com.google.common.base.Function;
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.Iterables;
+import static org.killbill.billing.util.collect.CollectionTransformer.iterableToList;
```

Comment:
Are we always consistent with static vs no-static import? Wondering also if there is a rule we could add in our IntelliJ `codeStyle` to help enforce this?

Reply:
Hmm.. in general AFAIK we're not.. :sweat_smile: . For example, I've seen some assertion do use static import, and some don't. Should we open discussion about this in slack?